app/models/review.rb

```diff
@@ -40,7 +40,7 @@ def recommended_reviewer?(user)
 
   # Is this review "recent"?
   def recent?
-    latest_revision_id = notebook.revisions.order(id: :desc).limit(1).pluck(:id).first
+    latest_revision_id = notebook.revisions.order(id: :desc).limit(1).map(&:id).first
```

Comment:
This should probably just be .first.id rather than .map.first.  the limit(1) and .first might be redundant anyway, research this